AppCenter/AppCenter/Internals/Util/MSJwtClaims.m

```diff
@@ -33,6 +33,19 @@ + (MSJwtClaims *)parse:(NSString *)jwt {
     return nil;
   }
   NSString *base64ClaimsPart = parts[1];
+
+  /*
+   * NSData's base64 parser expects the length to be padded with '=' to indicate the size of the last "chunk."
+   * Learn more here: https://tools.ietf.org/html/rfc4648#section-4.
+   */
+  int padding = 0;
```

Comment:
I have a feeling we already dealt with this base64 issue in other places.
Do we have a utils for that? Are we duplicating any code here?

Reply:
Looks references were msal code and our own docs for manual decoding, but that's a first in AppCenter.